ui/components/DashboardComponent.js

```diff
@@ -75,13 +75,17 @@ const styles = (theme) => ({
   },
   card: {
     height: '100%',
+    boxShadow: '0px 0px 10px grey',
```

Comment:
Please remove shadows, it is not looking well, the shadow in mockup was to demonstrate. Material-UI already has a minimal shadow component which is quite neat and tidy.

Reply:
Okay i will revert these changes.